docs/04-api-reference/front-commerce-remix/defineRemixExtension.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "defineRemixExtension"
+sidebar_position: 4
+description:
+  "defineRemixExtension allow you to extend the features of Front-Commerce for a
+  Remix application."
+---
+
+<p>
+  Extensions are registered in the application using the
+  front-commerce.config.ts file. Read [Register an
+  extension](/docs/remixed/guides/register-an-extension) to learn more.
+</p>
+
+## `defineRemixExtension`
+
+`defineRemixExtension` allow you to define extensions settings, this is extends
+the [`defineExtension`](../front-commerce-core/defineExtension) configuration of
+Front-Commerce.
+
+```js
+defineRemixExtension(configuration);
+```
+
+Arguments:
+
+- `configuration`
+  ([`RemixExtensionConfig`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/remix/config/defineRemixExtension.ts?ref_type=heads#L8)):
+  the extension configuration
+
```

Comment:
**suggestion(non-blocking):** we used "table" format for argument in other pages (i.e.: [FC-core/React](https://deploy-preview-796--heuristic-almeida-1a1f35.netlify.app/docs/remixed/api-reference/front-commerce-core/react#useconditionalcallonmount)). It may be nice to have it there as well.